packages/runner/src/recipe-manager.ts

```diff
@@ -11,6 +12,18 @@ export const recipeMetaSchema = {
     spec: { type: "string" },
     parents: { type: "array", items: { type: "string" } },
     recipeName: { type: "string" },
+    additionalSources: { 
+      type: "array", 
+      items: {
```

Comment:
is anything else needed for persistence, other than storing as a part of the JSON schema?

wonder where else we use the `src` (a few times for IDs i think)